engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -2485,6 +2485,10 @@ class EngineSemantics {
     final PlatformConfiguration newConfiguration = EnginePlatformDispatcher.instance.configuration
         .copyWith(accessibilityFeatures: original.copyWith(accessibleNavigation: value));
     EnginePlatformDispatcher.instance.configuration = newConfiguration;
+    Future.microtask(() {
+      // Avoid executing MediaQuery._updateData during build time
```

Comment:
This should be done in `MediaQuery` since web engine doesn't know (also shouldn't care ) about the build phase.

Reply:
I fixed code [322d936](https://github.com/flutter/flutter/pull/166836/commits/322d9368ff1776bf7859de04697a981c94e51db7), and update test cases [44d3b5b](https://github.com/flutter/flutter/pull/166836/commits/44d3b5b94c4e31a70e82b4ebd5754b812f62e0de)!